src/Microsoft.Tye.Core/DockerFileServiceBuilder.cs

```diff
@@ -13,6 +15,7 @@ public DockerFileServiceBuilder(string name, string image)
         }
         public string Image { get; set; }
 
+        public Dictionary<string, string> BuildArgs { get; set; } = new Dictionary<string, string>();
```

Comment:
Move below DockerFile to keep ordering consistent.

Reply:
done